features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -946,5 +946,7 @@
 
   "show_pending_user_information.enable": true,
   "identity_util.enable_sha256_random_numbers": true,
-  "extension_mgt.extension_types": "applications,identity-providers,identity-verification-providers"
+  "extension_mgt.extension_types": "applications,identity-providers,identity-verification-providers",
+
```

Comment:
is this blank line necessary?

Reply:
Default values are separated into sections. Since this is a new config, I added as a separate section. Can remove if it is redundant.

Reply:
ack, makes sense